src/SOS/SOS.UnitTests/Debuggees/WebApp/WebApp.csproj

```diff
@@ -6,7 +6,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.22"/>
```

Comment:
nit: Do you want to move this to Version.props?

Reply:
not for this one. It's a test project. I don't want to mix product and test versions if possible.